test/powershell/enginecore/Eventing.Tests.ps1

```diff
@@ -0,0 +1,36 @@
+Describe "BooleanParameterDCR Tests" -tags "CI" {

```

Comment:
it's a cut and paste from the other file - shouldn't it just be called "Event Subscriber tests"?